website/docs/d/route_table.html.markdown

```diff
@@ -44,6 +44,8 @@ The following attributes are exported:
 
 The `route` block exports the following:
 
+* `disable_bgp_route_propagation` - Boolean flag which controls propagation of routes learned by BGP on that route table.
```

Comment:
This is documented under the route block but in the schema it's defined as a top level property?

Reply:
Totally agree with you, thanks for pointing that out, I'll fix that.